subsys/bluetooth/host/hci_core.c

```diff
@@ -175,18 +175,17 @@ void bt_hci_host_num_completed_packets(struct net_buf *buf)
 	uint16_t handle = acl(buf)->handle;
 	struct bt_hci_handle_count *hc;
 	struct bt_conn *conn;
-
+	uint8_t index = acl(buf)->index;
```

Comment:
Nit: The Zephyr style is to have a blank like after declarations.

Reply:
fixed now